cmd/client.go

```diff
@@ -147,7 +147,7 @@ func (c *CloudproberClient) ListProbes(ctx context.Context) ([]*proberpb.Probe,
 	// Sorts the probes by their extension number as they are in the format:
 	// "testExtension" + <number>, e.g. "testExtension0".
 	sort.SliceStable(probesList[:], func(i, j int) bool {
-		probeNum0, _ := strconv.Atoi(probesList[i].GetName()[13:])
+		probeNum0, _ := strconv.Atoi(probesList[i].GetName()[13:]) // 13 because there are 13 characters in "testExtension" and we want the int ID.
```

Comment:
You can eliminate the comment by expressing it with code:
strconv.Atoi(probesList[i].GetName()[len("testExtension"):])

Reply:
Aha! Denia and I were just discussing this and how it could be made more legible. Thank you!